test_ak_survey_results.py

```diff
@@ -1,42 +1,42 @@
 import pytest
-import test_settings
+from test_settings import settings
```

Comment:
Are the test_settings actually in Github? Should they be?  We don't want to use any real passwords of course but schema name and so on should be okay.

Reply:
The `test_settings.py` file is not in the repo; the README suggests creating it by copying `settings.py.example`. It's also in the `.gitignore`, so I think this is pretty safe.